src/activity_item_handling.cpp

```diff
@@ -2290,6 +2290,8 @@ void activity_on_turn_move_loot( player_activity &act, Character &you )
                                 continue;
                             }
                             you.gunmod_remove( *it->first, *mod );
+                            // need to return so the activity starts
+                            return;
                             move_item( you, *mod, 1, src_loc, src_loc, this_veh, this_part );
```

Comment:
This line and the line below are now unreachable. Why are they still there?

Reply:
Oversight.